Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+django = "*"
+psycopg2-binary = "*"
```

Comment:
I found that changing the psycopg2-binary version to be explicitly 2.9.2 made it work for me, not sure if that was an issue for you.